tests/test_helpers.py

```diff
@@ -115,6 +115,14 @@ class Foo(pyvista.PolyData):
         pyvista._wrappers = default_wrappers  # always reset back to default
 
 
+def test_wrap_no_copy():
+    # makes sure that wrapping an already wrapped object returns source
+    mesh = pyvista.Wavelet()
+    wrapped = pyvista.wrap(mesh)
+    assert mesh == wrapped
+    assert id(mesh) == id(wrapped)
```

Comment:
Then should we have this test to verify that rule of thumb? Or maybe this is captured elsewhere in the tests...

Reply:
No, this is a python thing. `id` tells you what `is` can when both objects still exist, and it misleads you otherwise. Just check object identity as in the suggestion.

Reply:
Ahh, thanks for the calirifcations